deploy/lib/server_config.rb

```diff
@@ -652,17 +652,24 @@ def bootstrap
 
     internals = find_arg(['--replicate-internals'])
     if internals
+      dointernals = 'internals'
 
+      # Number of hosts
+      r = execute_query %Q{ fn:count(xdmp:hosts()) }
+      r.body = parse_body(r.body)
+
+      # check cluster size
       nr = find_arg(['--nr-replicas'])
       if nr
-        nr = nr.to_i
+        if nr == "max" or nr == "MAX"
+          nr = r.body.to_i - 1
+        else
+          nr = nr.to_i
+        end
       else
         nr = 2
```

Comment:
Wouldn't 1 be a better default number of replicas since that is the minimum number of replicas allowed? (according to line 673..)